latex-gitCheatSheet/FSFW_git-Spickzettel.tex

```diff
@@ -94,162 +97,165 @@
+	\subsection{Branch B1 in B2 mergen (dt. mischen)}
 		\cmdline{git checkout <B2>}
 		\cmdline{git merge <B1>}
-                \annotation{or}
+                \annotation{oder}
 		\cmdline{git merge -{}-no-ff <B1>}
-		\annotation{generates commit even if fast-forwarding is possible}
-	\subsection{Create Branch based on HEAD and checkout}
+		\annotation{Erstelle einen Commit auch wenn fast-forwarding möglich ist}
+	\subsection{Erstellen eines Branches basierend auf HEAD und checkout}
 		\cmdline{git checkout -b <branch>}
-	\subsection{Delete a branch}
+	\subsection{Löschen eines Branches}
 		\cmdline{git branch -d <branch>}
 		\cmdline{git branch -D <branch>}
-                \annotation{Delete a branch that is not merged to the default branch}
+                \annotation{Lösche einen, noch nicht mit dem Standard-Branch gemergeten, Branch}
 }
 
 
 %% how to update repository
 \colouredbox{pink}{
-	\begin{center}\section{Get changes from upstream}\end{center}
-	\subsection{Fetch changes from a remote}
+	\begin{center}\section{Änderungen von entferntem Repos holen}\end{center}
+	\subsection{Lade Änderungen von remote *(einem entfernten Repo) herunter}
 		\cmdline{git fetch [remote]}
-                \annotation{\texttt{remote} defaults to \texttt{origin}}
-        \subsection{Get changes}
+                \annotation{\texttt{remote} ist voreingestellt auf: \texttt{origin}}
+        \subsection{Änderungen bekommen}
                 \cmdline{git pull [remote] [refspec]}
                 \annotation{\texttt{remote} defaults to \texttt{origin}}
 }
 
 
 %% Resolve merge conflicts
 % \colouredbox{gray}{
-% 	\begin{center}\section{Resolve merge conflicts}\end{center}
-% 	\subsection{View merge conflicts}
+% 	\begin{center}\section{Lösen von Merge-Konflikten}\end{center}
+% 	\subsection{Merge-Konflikte anzeigen}
 % 		\cmdline{git diff}
-% 	\subsection{View merge conflicts against base file}
+% 	\subsection{Zeige Merge-Konflikte gegen Basisdateien}
 % 		\cmdline{git diff -{}-base <FILE>}
-% 	\subsection{View merge conflicts against other changes}
+% 	\subsection{Zeige Merge-Konflikte gegen Änderungen Anderer}
 % 		\cmdline{git diff -{}-theirs <FILE>}
-% 	\subsection{View merge conflicts against your changes}
+% 	\subsection{Zeige Merge-Konflikte gegen deine Änderungen}
 % 		\cmdline{git diff -{}-ours <FILE>}
-% 	\subsection{After resolving conflicts, merge with}
+% 	\subsection{Nach dem Auflösen von Konflikten committen nicht vergessen}
 % 		\cmdline{git add <CONFLICTING\_FILE>}\\
 %                 \cmdline{git commit}
 % }
 
 %% Publish your changes
 \colouredbox{blue}{
-	\begin{center}\section{Publishing changes}\end{center}
-	\subsection{Push changes to a remote}
+	\begin{center}\section{Änderungen veröffentlichen}\end{center}
+	\subsection{Änderungen nach remote pushen (also in ein entferntes Repo schieben)}
 		\cmdline{git push [origin] [branch]}
-	\subsection{Create tags}
+	\subsection{Einen tag (dt.: ???) erstellen}
```

Comment:
Einfach bei "Tag" belassen, deutsche Übersetzung weglassen. "Tag" ist ja ein relativ eingedeutschter Begriff heutzutage. #thankstwitter

Reply:
Ich bin für das Genus Neuter für "Tag" (unter anderem um es von "der Tag" zu unterscheiden), [der Duden ist meiner Meinung](http://www.duden.de/rechtschreibung/Tag_Strukturelement_Markierung).

Reply:
Grummelgrummel, jaja… Damit werde ich mich nicht anfreunden können ;-).

Reply:
Ich bin dafür es mir Fragezeichen zu belassen. Schließlich ist es work-in-progress und Mitmachen erwünscht.